UnityProject/Assets/Scripts/UI/Objects/Research/GUI_ArtifactAnalyser.cs

```diff
@@ -0,0 +1,235 @@
+using System.Collections;
+using UnityEngine;
+using UI.Core.NetUI;
+using Systems.Research.Objects;
+using Items.Science;
+
+namespace UI.Objects.Research
+{
+	public class GUI_ArtifactAnalyser : NetTab
+	{
+		[SerializeField]
+		private NetPageSwitcher mainSwitcher = null;
+		[SerializeField]
+		private NetPage scanPage = null;
+		[SerializeField]
+		private NetPage noSamplePage = null;
+
+		[SerializeField]
+		private NetText_label rpLabel = null;
+		[SerializeField]
+		private NetSpriteImage connection = null;
+
+		[SerializeField]
+		private NetText_label scanStatusLabel = null;
+
+		//Results
+		[SerializeField]
+		private NetText_label radLabel = null;
+		[SerializeField]
+		private NetText_label bluespaceLabel = null;
+		[SerializeField]
+		private NetText_label clownLabel = null;
+		[SerializeField]
+		private NetText_label massLabel = null;
+		[SerializeField]
+		private NetText_label completeLabel = null;
+
+		public ArtifactAnalyser artifactAnalyser;
```

Comment:
Public fields are automatically serialised by Unity. If you want it to be a public field and serialisation doesn't make sense, please add the `[NonSerialized]` attribute.

```suggestion
		public ArtifactAnalyser artifactAnalyser { get; private set; }
```
(naming convention would mean this'd be called `ArtifactAnalyser` too)

There's a bunch of instances like this, would be most appreciated if you could check all your public fields for your classes and correct 🙂